test/integration/scenarios/network/peers_blacklist.js

```diff
@@ -76,30 +72,40 @@ module.exports = params => {
 				);
 			});
 
-
 			describe('when a node blacklists an ip', () => {
 				before(done => {
+					// By default, current devnet configurations don't have all the parameters
+					// A pre-control is needed to avoid exceptions
+					if (!('access' in params.configurations[0].peers)) {
+						params.configurations[0].peers.access = {};
+					}
+					if (!('blackList' in params.configurations[0].peers)) {
+						params.configurations[0].peers.access.blackList = [];
+					}
 					params.configurations[0].peers.access.blackList.push('127.0.0.1');
 					fs.writeFileSync(
 						`${__dirname}/../../configs/config.node-0.json`,
 						JSON.stringify(params.configurations[0], null, 4)
 					);
-					restartNode('node_0');
+					// Restart the node to load the just changed configuration
+					common.restartNode('node_0');
 					setTimeout(() => {
 						blockchainReady(done, null, null, 'http://127.0.0.1:4000');
-					}, 4000);
+					}, 8000);
 				});
 
-
-				it(`there should be ${expectedConnectionsAfterBlacklisting} established connections from 500[0-9] ports`, done => {
+				it(`there should be ${EXPECTED_OUTOGING_CONNECTIONS_AFTER_BLACKLISTING} established connections from 500[0-9] ports`, done => {
 					utils.getEstablishedConnections(
 						Array.from(wsPorts),
 						(err, numOfConnections) => {
 							if (err) {
 								return done(err);
 							}
 
-							if ((numOfConnections - 20) <= expectedConnectionsAfterBlacklisting) {
+							if (
```

Comment:
That test is not reliable - it will pass when there is 0 connections.

Reply:
There are many tests like that here. Why? We know exactly how many nodes we have, we know exactly how many open connections should be there, so why not strict check?

Reply:
Same as above.